src/help.py

```diff
@@ -37,18 +36,9 @@
 """
 )
 
-temp_exporter = Exporter()
```

Comment:
Since now `Exporter` does not contain the counters we do not have to instantiate a temporary object just to get their info which should be good :).